### ISSUE TITLE
Installing "PSMatch" package

### ISSUE THREAD

**Issue by HijaziHassan** [rformassspectrometry/PSMatch#10]
Hello Laurent,
I faced a problem while trying to install PSMatch package.
`BiocManager::install("RforMassSpectrometry/PSMatch")`
(adding `dependencies = TRUE` didn't help as well)

![PSMatch  error](https://user-images.githubusercontent.com/57935816/145731489-09480ecc-f3c4-4ab5-b365-eee93ded6b13.png)

I then realized that the package is no more here
[https://bioconductor.org/packages/3.15/PSMatch/](url)

Could you please help here.

**Comment by lgatto** [comment]
You need the latest development version of `ProtGenerics` which you get using R-devel and/or from [Github](https://github.com/Rformassspectrometry/ProtGenerics).

**Comment by HijaziHassan** [comment]
following your advise,
I used this command `.libPaths()` to check where my libraries  are found. There, I deleted the folders named "Matrix" and "ProtGenerics" then  restarted `R`. 
After that, I downloaded `ProtGenerics` and `PSMatch`.  The `Matrix`, being one of the dependencies, was downloaded and took a while and many output lines to be installed.

I used those two lines of code after installing the `devtools` package:

`devtools::install_github("RforMassSpectrometry/PSMatch")`
`devtools::install_github("Rformassspectrometry/ProtGenerics")`

Thanks again. I appreciate it, Laurent.
